src/python/gem5/resources/client_api/atlasclient.py

```diff
@@ -102,6 +103,7 @@ def _atlas_http_json_req(
         data_json: Dict[str, Any],
         headers: Dict[str, str],
         purpose_of_request: Optional[str],
+        proxy_context: Optional[object] = None,
```

Comment:
Type should be https://docs.python.org/3/library/ssl.html#ssl.SSLContext, I think